api/src/main/java/bio/terra/tanagra/service/search/Relationship.java

```diff
@@ -14,6 +14,37 @@
   /** The second entity in the relationship. */
   public abstract Entity entity2();
 
+  /**
+   * Returns whether the {@code x} and {@code y} entities are the two entities of the relationship,
+   * in any order.
+   */
+  public boolean unorderedEntitiesAre(Entity x, Entity y) {
```

Comment:
nit: are what? Perhaps it's meant to be implied by the `Relationship` noun, but I don't quite follow. Maybe something like `relatesEntitiesUnordered(x, y)` ? Or just `hasEntitiesUnordered(x, y)` for consistency with the other method?

Reply:
thanks, went with `hasEntitiesUnordered`